packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -33,6 +33,7 @@ template <typename DeviceType>
 class BVH
 {
   public:
+    BVH() = default;
```

Comment:
Add a comment to say that this constructor should not be used since nothing useful can be done with this constructor.